vscode/src/ruby.ts

```diff
@@ -19,6 +20,7 @@ export enum ManagerIdentifier {
   Rvm = "rvm",
   Shadowenv = "shadowenv",
   Mise = "mise",
+  Windows = "windows",
```

Comment:
I wonder if we should be more specific here, e.g. `RubyInstallerWindows`.

Reply:
My idea with going less specific is that if there's another popular way of installing Ruby on Windows we discover later, then we can just include it in the same implementation.

But I'm fine with making it more specific too. Do you prefer that?

Reply:
Renamed to `RubyInstaller`.